docs/training/training-101.adoc

```diff
@@ -0,0 +1,61 @@
+= Beer Garden 101
+
+:page-layout: docs
+:includedir: _includes
+:guide_dir: ../python
+
+This course is designed to take a developer who is familiar with Python and
+bring them up to speed on how to create and execute a Plugin on Beer Garden
+
+== Overview of Beer Garden
+
+Beer Garden is targeted to help developers break that barrier between python
+scripts to software. So much of software development is tangled in the weeds
+of standing up services, getting messages from one component to another,
+creating UIs, writing documentation, and concurrently supporting multiple
+versions. We wanted to change that. Beer Garden makes it as easy as running
+the main method.
+
+Development of the Beer Garden framework is written in Python for Python. This has been a
+multi-year effort to bring Beer Garden to that state it is today. It is constantly
+evolving as we learn new and innovative techniques. It also evolves with the each new
+contributor because everyone brings their own unique skills and passions.
+
+include::../what-is-beergarden.adoc[tags=how-can-it-help]
+
+== Before we Start
+
+include::../python/_includes/prereqs.adoc[]
+
+include::../what-is-beergarden.adoc[tags=terminology]
+
+include::ui-orientation.adoc[]
+
+== Writing Plugins for Beer Garden
+
+This is a guided plugin development that are intended to be deployed against an active Beer Garden
+instance. This only focuses on the developing remote plugins.
+
+link:../local-vs-remote.adoc[Local vs Remote]::
+If you are interested in the difference between Remote and Local plugins, check out our guide
```

Comment:
I'd honestly just take this part about local vs remote plugins out. I've always meant to move this out of the initial flow on the site. It's not something that needs to be noted until after people understand what a plugin really is. Focusing on it before actually doing the example is just going to confuse people and they're not going to retain the distinction.